.changeset/grumpy-gifts-end.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": patch
+---
+
+feat: `wrangler secret * --local`
+
+This PR implements `wrangler secret` for `--local` mode. The implementation is simply a no-op, since we don't want to actually write secret values to disk (I think?). I also got the messaging for remote mode right by copying from wrangler 1. Further, I added tests for all the `wrangler secret` commands.
```

Comment:
I'm not sure I follow the reasoning behind not storing to disk. The current workflow for `secret` is run it once, then never need to again. If that workflow differs for `--local` it might cause more complication. (If you have a Worker with a secret, how do you test it locally?)

Reply:
I'm actually not sure what to do here! Writing to disk introduces the chance of people committing it to git or a cache, which may be worse. I'm not sure what the workflow here should be, and it probably requires a resolution on https://github.com/cloudflare/wrangler2/issues/190 to move forward. minflare doesn't even implement a workflow for secrets. This PR would at least have _something_, is why I opened it up. Got any ideas?

Reply:
ping

Reply:
Hmm, interesting. I guess let's merge for now and figure that out later.